Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1511,13 +1510,13 @@ public class BrowserViewController: UIViewController {
   override public func observeValue(forKeyPath keyPath: String?, of object: Any?, change: [NSKeyValueChangeKey: Any]?, context: UnsafeMutableRawPointer?) {
 
     guard let webView = object as? WKWebView else {
-      log.error("An object of type: \(String(describing: object)) is being observed instead of a WKWebView")
+      Logger.module.error("An object of type: \(String(describing: object), privacy: .public) is being observed instead of a WKWebView")
       return  // False alarm.. the source MUST be a web view.
     }
 
     // WebView is a zombie and somehow still has an observer attached to it
     guard let tab = tabManager[webView] else {
-      log.error("WebView: \(webView) has been removed from TabManager but still has attached observers")
+      Logger.module.error("WebView: \(webView, privacy: .public) has been removed from TabManager but still has attached observers")
```

Comment:
Will this log an URL? Do we need to make it public?

Reply:
It should print out memory address only, but makes no sense to risk it keeping public, will remove.

When I first started on this project i was deciding on whether to keep stuff public or private per each log line.
But then the changes were so big that i decided to make everything private by default and make things public as we go instead of evaluating each line